md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/HiveTableMigrationAPI.java

```diff
@@ -0,0 +1,250 @@
+package com.wipro.ats.bdre.md.rest.ext;
+
+import com.wipro.ats.bdre.md.dao.ProcessDAO;
+import com.wipro.ats.bdre.md.dao.jpa.Properties;
+import com.wipro.ats.bdre.md.rest.RestWrapper;
+import com.wipro.ats.bdre.md.rest.RestWrapperOptions;
+import com.wipro.ats.bdre.md.rest.util.Dao2TableUtil;
+import com.wipro.ats.bdre.md.rest.util.DateConverter;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by SU324335 on 3/29/2016.
+ */
+@Controller
+@RequestMapping("/hivemigration")
+public class HiveTableMigrationAPI {
+
+
+    private static final Logger LOGGER = Logger.getLogger(HiveTableMigrationAPI.class);
+    private static final String HIVEMIGRATION = "hive-migration";
+    private static Connection connection;
+    private static String driverName = "org.apache.hive.jdbc.HiveDriver";
+
+
+    @Autowired
+    private ProcessDAO processDAO;
+
+    //Fetching all the databases from hive of source cluster
+    @RequestMapping(value = "/databases/{srcEnv:.+}", method = {RequestMethod.GET})
+    @ResponseBody
+    public RestWrapperOptions getDBList(@PathVariable("srcEnv") String sourceEnv) {
+
+        RestWrapperOptions restWrapperOptions = null;
+        LOGGER.info(sourceEnv + "srcEnv");
+        try {
+            Class.forName(driverName);
+            connection = DriverManager.getConnection("jdbc:hive2://" + sourceEnv + "/default", "", "");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "jdbc:hive2://" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)